src/modules/70-pipeline/components/PipelineSteps/AdvancedSteps/FailureStrategyPanel/FailureStrategyPanel.tsx

```diff
@@ -38,10 +44,17 @@ export interface FailureStrategyPanelProps {
   mode: Modes
   isReadonly: boolean
   stageType?: StageType
+  isStepInsideStepGroup?: boolean
 }
 
 export function FailureStrategyPanel(props: FailureStrategyPanelProps): React.ReactElement {
-  const { mode, path = 'failureStrategies', isReadonly, stageType = StageType.DEPLOY } = props
+  const {
+    mode,
+    path = 'failureStrategies',
+    isReadonly,
+    stageType = StageType.DEPLOY,
+    isStepInsideStepGroup = 'false'
```

Comment:
```suggestion
    isStepInsideStepGroup = false
```
This is of boolean type